### DISCUSSION TITLE
Github Pages with a SquareSpace custom domain

### DISCUSSION THREAD

**Discussion by raleighlittles** [community/community#81779] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I've set up Github Pages sites before using Google Domains, and now I'm trying to set one up with SquareSpace (since there's no more Google Domains), but I'm running into DNS issues.

Has anyone had this happen recently?

Following the Github instructions here: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain-and-the-www-subdomain-variant

I created a total of 9 new records: 4 "A" records, 4 "AAAA" records, and a CNAME record pointing from my domain to `raleighlittles.github.io`.

Squarespace DNS screenshot: https://i.imgur.com/nVsK8vw.png

![image](https://github.com/community/community/assets/2137201/9da983ba-e1a7-40d6-a9ca-0b5cb002ce9e)

In the Github repository settings, I'm trying to have it use the new domain, but I'm getting this error: https://i.imgur.com/4F3IJlS.png

![image](https://github.com/community/community/assets/2137201/86903e4f-0a32-4a00-b136-9f65841ad89f)

> Both www.raleighlittles.games and its alternate name are improperly configured
> Domain's DNS record could not be retrieved. For more information, see [documentation] (https://docs.github.com/articles/setting-up-a-custom-domain-with-github-pages/) (InvalidDNSError).

Weirder yet if I search the documentation for "InvalidDNSError" nothing comes up.

I figured I'd check here before filing a SquareSpace support ticket in case it's something specific for Pages.

**Comment by yoannchaudet** [comment]
Any DNS change can take some time to propagate everywhere on the Internet. Looking at `raleighlittles.games` now, it is correctly pointing at GitHub Pages and I see you have a valid TLS certificate for it. So issue solved!

**Comment by bendrice** [comment]
What did you end up changing in your DNS in squrespace to get this to work?

**Reply by byronthe123** [reply]
See this article: https://emilymdubois.medium.com/using-a-squarespace-domain-with-github-pages-438951d4f5b7
Also, make sure that in github the custom domain is mapped as `www.my-domain.com` with the `www.`

**Comment by raleighlittles** [comment, marked as answer]
I'm sorry I forgot to come back to this post. The problem I had was that I shouldn't have been using the full website name and instead use the "@" variable as the _Host_.

Here are my current Squarespace settings that work

![image](https://github.com/community/community/assets/2137201/0b9f2e09-b177-43bd-a2bf-c3a1da15258e)

**Reply by bendrice** [reply]
Quick question, what are your AAAA type records? My site still wont publish and I don't have those in my records - assuming that might be the problem for me.

**Reply by raleighlittles** [reply]
Those are for IPv6 only, the A records are for IPv4. The IP addresses come from Step 5 of here https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain

**Comment by hmpsmathuho** [comment]
![image](https://github.com/community/community/assets/174812842/ad61c8c5-7321-4fc6-80be-9f414fcffff9)
![image](https://github.com/community/community/assets/174812842/923335ee-8000-4725-a18b-801a9eb30cce)
i alredy follow the step above but still got error

**Reply by yoannchaudet** [reply]
I see your certificate has already been approved since you posted this message.

**Comment by mehdiataei** [comment]
Same issue. Did you keep or delete the "Squarespace Domain Forwarding"?
